docs/README.md

```diff
@@ -0,0 +1,58 @@
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![python](https://img.shields.io/badge/Python-3.11-3776AB.svg?style=flat&logo=python&logoColor=white)](https://www.python.org)
+
+# AI-on-Demand 
+
+The [AI-on-Demand](https://aiod.eu) (AIOD) platform empowers AI research and innovation for industry and academia. 
+At its core if the [metadata catalogue](https://aiod.i3a.es) which indexes countless AI resources, such as datasets, papers, and educational material, 
```

Comment:
I think you meant (https://api.aiod.eu), right?

Reply:
Yep! Fixed.